### PR TITLE
Mm/calendar view

### PR DESCRIPTION
**Pull request by MissyM2**
Added navigation and formatting of components

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (meissadia) [inline .vscode/settings.json]: Add the .vscode directory to the .gitignore

File: src/components/CashAvailableMonth.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <div className="page-subtotal">Total Available Cash</div>
+    <div className="total-available-cash-value-wrapper">
+      <div className="dollar-sign-icon">
+        <svg
+          aria-labelledby="title"
+          width="16"
+          height="16"
+          fill="#1579d0"
+          xmlns="http://www.w3.org/2000/svg"
+          // xmlns:xlink="http://www.w3.org/1999/xlink"
```
Comment (meissadia) [inline src/components/CashAvailableMonth.jsx]: Delete comment

File: src/components/CurrentMonthSummary.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import CurrentMonthSummaryChart from "./CurrentMonthSummaryChart";
+
+const CurrentMonthSummary = () => (
+  <div className="monthly-totals-wrapper">
+    <div className="monthly-totals-header">
+      <div className="page-subtitle">Monthly Totals</div>
+      <div className="show-link">
+        <Link to="/currentmonthsummarychart">Show</Link>
+        <div>
+          <svg
```
Comment (meissadia) [inline src/components/CurrentMonthSummary.jsx]: import { ReactComponent as IconName } from '../assets/icon.svg';

<IconName className='trashIcon' />

// CSS
div svg {
  fill: white;
}

File: .vscode/settings.json (hunk 1 above)
Comment (meissadia) [reply .vscode/settings.json]: Actually, you can delete this file and set the 'Format on save' in the VS code settings.

Comment (meissadia) [reply .vscode/settings.json]: Command + ,   => Open settings

Type 'format' in the 'Search settings' box

Check the box for 'Editor: Format On Save'

File: src/components/CurrentMonthSummary.jsx (hunk 3 above)
Comment (MissyM2) [reply src/components/CurrentMonthSummary.jsx]: I changed all the svgs to React Components

File: src/components/CashAvailableMonth.jsx (hunk 2 above)
Comment (MissyM2) [reply src/components/CashAvailableMonth.jsx]: done

**Pull request merged**